test_runner/src/main/kotlin/ftl/args/ArgsHelper.kt

```diff
@@ -265,6 +265,20 @@ object ArgsHelper {
 
         return shards.map { Chunk(find + it.testMethods.filterNot { method -> find.contains(method) }) }
     }
+
+    fun calculateDummyShards(
```

Comment:
Do we really need 2 methods named `calculateDummyShards` which both have only one call? IMO It's unnecessary redundancy. Besides that `ArgHelper` will be refactored soon.

Reply:
I believe the intention was to be in line with the current implementation https://github.com/Flank/flank/blob/733e444bc4db1b80c9117c409c7ecec898f2c344/test_runner/src/main/kotlin/ftl/run/platform/android/CreateAndroidTestContext.kt#L49-L52
As you mentioned, there will be a huge refactor soon so within this PR we either refactor all similar occurrences or leave it as it is

Reply:
removed the reference from ArgsHelper 👍

Reply:
@pawelpasterz I removed the reference from ArgsHelper. and merely moved the logic into the CreateAndroidTestContext